src/randoop/sequence/Sequence.java

```diff
@@ -839,9 +843,11 @@ public boolean isOwnerOf(Variable v) {
    * Returns a string representing this sequence. The string can be parsed back
    * into a sequence using the method Sequence.parse(String). In particular, the
    * following invariant holds:
-   *  <pre>st.equals(parse(st.toParseableCode()))</pre>
-   *  See the {@link randoop.Sequence#parse(List) parse} for the 
+   *  <code>st.equals(parse(st.toParseableCode()))</code>
```

Comment:
I would have left it as `<pre>`, but if you are change it to `<code>` then add a period at the end of the sentence.